tests/jeos/build_key.pm

```diff
@@ -14,18 +14,25 @@ use strict;
 use warnings;
 use base "opensusebasetest";
 use testapi;
+use version_utils "is_opensuse";
 
 sub run {
     # check that suse-build-key is installed
     my $zypper_output = script_output('zypper search suse-build-key');
-    die 'suse-build-key is not installed' unless $zypper_output =~ /i\+(.*)suse-build-key/;
 
-    assert_script_run('ls /usr/lib/rpm/gnupg/keys | grep suse_ptf_key.asc');
+    if (is_opensuse) {
+        die 'suse-build-key is not installed' unless $zypper_output =~ /i\+(.*)openSUSE-build-key/;
+        assert_script_run('ls /usr/lib/rpm/gnupg/keys | grep .asc');
```

Comment:
Note that this is already tested by installing packages - if `zypper in <some package>` works, the key is there.

So IMO testing for files in `/usr/lib/rpm/gnupg/keys` is not necessary.

Reply:
Updated.
New validation runs: http://ccret.suse.cz/tests/2912, http://ccret.suse.cz/tests/2913